code/src/components/TaskSummary.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+
+
+export const TaskSummary = () => {
+  const items = useSelector(state => state.tasks.items)
+  const leftTodo = items.filter(item => !item.complete)
+  const totalTodo = items.length
+
+  const [showLeftTodo, setShowLeftTodo] = useState(false)
+
+  return (
+    <SummaryWrapper>
+      <Input>
```

Comment:
Is input really a good name for this styled component? It's not really an input right?